vocabularies/Hierarchy.md

```diff
@@ -13,10 +13,20 @@ Term|Type|Description
 ## <a name="RecursiveHierarchyType"></a>[RecursiveHierarchyType](./Hierarchy.xml#L46:~:text=<ComplexType%20Name="-,RecursiveHierarchyType,-") *([Experimental](Common.md#Experimental))*
 
 
+The nullable property paths in this complex type refer to properties of an entity type that contain
+          information about a node in a recursive hierarchy. When a resource of the annotated entity type is requested
+          and hierarchy-related transformations (`ancestors`, `descendants`, `traverse`) applied using the hierarchy qualifier of the annotation,
```

Comment:
```suggestion
          and hierarchy-related transformations (`ancestors`, `descendants`, `traverse`) applied to it using the resource as first parameter and the hierarchy qualifier of the annotation as second parameter,
```

I read your description as if you apply the simplification/restriction we discussed in our call: The first parameter of ancestors/descendants is identical the input set. If that's what you intended I would state it clearly as suggested